modular_zubbers/code/modules/antagonists/uplink/uplinkdatums.dm

```diff
@@ -7,3 +7,8 @@
 	cost = 8
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_tc/henchmen_traitor_outfits
+	name = "Henchmen Bundle"
+	desc = "A set of five armored henchmen outfits! Each set comes with a cap, coat, uniform, gloves, shoes, and a switchblade!"
+	item = /obj/item/storage/backpack/duffelbag/henchmen_traitor_outfits
+	cost = 4
```

Comment:
This should probably be under the badass category instead

Reply:
Done.